tools/accuracy_checker/accuracy_checker/config/config_validator.py

```diff
@@ -568,6 +568,47 @@ def parameters(self):
         return parameters_dict
 
 
+class NormalizationArgsField(BaseField):
+    PRECOMPUTED_MEANS = {
+        'imagenet': (104.00698793, 116.66876762, 122.67891434),
+        'cifar10': (125.307, 122.961, 113.8575),
+    }
+
+    PRECOMPUTED_STDS = {
+        'imagenet': (104.00698793, 116.66876762, 122.67891434),
+        'cifar10': (125.307, 122.961, 113.8575),
+    }
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+
+    def validate(self, entry, field_uri=None, fetch_only=False, validation_scheme=None):
+        error_stack = super().validate(entry, field_uri, fetch_only, validation_scheme)
+        if entry is None:
+            return error_stack
+
+        field_uri = field_uri or self.field_uri
+
+        precomputed_args = NormalizationArgsField.PRECOMPUTED_MEANS if 'mean' in field_uri else (
+            NormalizationArgsField.PRECOMPUTED_STDS)
+
+        entry = get_or_parse_value(entry, precomputed_args)
+
+        if 'std' in field_uri and 0 in entry:
```

Comment:
also more general way, if you add parameter allow_zeros=True/False

Reply:
Added